source/Modules/PDT/PDT.psm1

```diff
@@ -745,7 +745,7 @@ function Wait-Win32ProcessStart
         [PSCredential] $Credential,
 
         [Parameter()]
-        [Int] $Timeout = 60000
+        [Int] $Timeout = 120000
```

Comment:
It would be great to bring this to the DSC community standard at some point.
So:
```suggestion
        [int]
        $Timeout = 120000
```

Reply:
Also the PDT.PSM1 should probably be moved to private functions.